contracts/prize-pool/PrizePool.sol

```diff
@@ -285,6 +285,8 @@ abstract contract PrizePool is OwnableUpgradeSafe, BaseRelayRecipient, Reentranc
   /// @param to The address the tickets are being transferred to
   /// @param amount The amount of tickets being trasferred
   function beforeTokenTransfer(address from, address to, uint256 amount) external override {
+    require(isControlled(_msgSender()), "PrizePool/uncontrolled-token");
```

Comment:
We already have a modifier called `onlyControlledToken`- you can just use that and pass in msg.sender

Reply:
right, was using _msgSender() when I don't need to